SPECS/dhcp/dhcp.spec

```
@@ -38,7 +41,13 @@ The ISC DHCP Client, dhclient, provides a means for configuring one or more netw
 
 
 %prep
-%autosetup -p1
+%setup -q -n dhcp-%{version}
```

I was curious why you had to revert back to `%setup` and realized I've made a mistake in my initial suggestion by forgetting about the `-N` flag to disable auto patching. I've tested this to work locally:
```suggestion
%autosetup -N
```

Sorry for the trouble.

---

Documentation about `%autosetup` if you're curious: https://rpm-software-management.github.io/rpm/manual/autosetup.html.